### ISSUE TITLE
Social Auth page OAUTH_CONFIGS syntax errors

### ISSUE THREAD

**Issue by tylorbayer** [tethysplatform/tethys#578]
In the documentation for Social Authentication, the LinkedIn and Hydroshare sections both have "-" before each item listed under the OAUTH_CONFIGS: setting. This throws an error when trying to start tethys. It took a while for me to look at the other social auths on the page to see how their oauth_configs were formatted.

**Comment by swainn** [comment]
@tylorbayer thanks for reporting. We've recently merged a commit into master to address several issues with the portal configuration documentation. Can you review on the "Latest" version of the docs and let me know if the issue still exists?

**Comment by tylorbayer** [comment]
Oh yeah it looks fixed there. I was actually wondering how to use the tethys settings command to set the auth configs. That is what it shows in the "latest" version. I tried to switch to "latest" from the social auth page yesterday to see if it was fixed in more recent documentation but the page didn't exist and I didn't think to go back to the main page and search for social auth in the docs again. Must have changed locations.  

Thanks.
